ovos_i2c_detection/__init__.py

```
@@ -53,3 +54,19 @@ def is_adafruit_amp():
+    if is_wm8960():
+        try:
+            ser = serial.Serial("/dev/serial0", 9600, timeout=3)
+            ser.write(b'system.version')
+            while True:
+                is_mk1 = ser.readline().decode().rstrip()
+                if is_mk1 and "Command" in is_mk1:
+                    # This is a Mark 1
+                    return True
+                break
+            return False
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)